### ISSUE TITLE
Operations [Renumeration]: Team Task Payments for March/April. Installment #1

### ISSUE THREAD

**Issue by wolstaeb** [C3ETH/c3eth-operations#193]
As per our consensus and #102 and #152, pay @C3ETH/c3eth-team task-based remuneration for March and April 2022. Perform the payment in one transaction for transparency and accountability purposes.

- [x] #102 and #152
- [x] Get the 7-Day Average USD/ADA price from [coinmarketcap.com](https://bit.ly/3fVrYKT). See details below.
- [x] [Update task compensation spreadsheet](https://docs.google.com/spreadsheets/d/1SHc84F-JPSyUSeylJ-JcfnxtqNUbYUZ0vNE97JF4g0c/edit?usp=sharing) with exchange rate, and
- [x] Work out Ada/Lovelace amounts, update script and double check amounts.
- [x] [Transfer total amount needed](https://explorer.cardano.org/en/transaction?id=97ffd8c83ffbf8b665ba24a2b9c8b3e2cdafb9b0c57342dff4fbc7d24571c39d) to the local payment address.
- [x] Update script with input transaction details
- [x] [Perform the renumeration payment](https://explorer.cardano.org/en/transaction?id=1e005ca93e3cf42a72b3c7c9067e235ce4f3cc3df402ba14b4627acf8361ce08)
- [x] [Notify the @C3ETH/c3eth-team of the transaction](https://discord.com/channels/868850218995707914/875343499824889947/977545220415692820).
- [x] Backup payment script details.
- [x] Document transaction details

---
## Transaction Details

Links above document the payment plan, the list of tasks being compensated, and the two associated on chain transactions (transfer/payment).

### Coin marketcap 7-Day rate

For Payment we use a simple seven day average to devise the payment conversion rates at the approximate time (within 30mins of) the payments being made. The USD/ADA rate is calculated using the following data points from Coinmarket Cap 7D prices. The intention is to smooth out fluations in the USD/ADA exchange rates.

```
ADA-7DA = (7DOPEN + 7DCLOSE + HIGH + LOW) / 4
```
Where: 7DOPEN, 7DCLOSE, HIGH, LOW are USD/ADA prices. HIGH and LOW are the Coinmarket Cap 7d Low/High, and the 7DSTART and 7DCLOSE are taken from the seven day window of the chart below.

PERIOD | 7DOPEN | 7DCLOSE | 7DLOW | 7DHIGH | AVG
-- | -- | -- | -- | -- | --
May | 0.5328 | 0.5322 | 0.6073 | 0.493 | 0.541325

![Screen Shot 2022-05-21 05 59 31 PM](https://user-images.githubusercontent.com/82072426/169640328-3a9a1233-e9b1-4a88-8921-bef2dbad6537.png)

## Installment Payments 1

ADA  price has drop siginificantly (as have Cryptocurrencies/Token prices across the board) over the last three weeks. Hence some blanacing of the wallet is needed. Also %50 of Fund7 funding for C3ETH is still due from the Catalyst Treasury. These are dependant on submission of close-out reports. 

![C3ETH-Team-Task-Payments-March-April-2022](https://user-images.githubusercontent.com/82072426/169677409-cdddb7cc-dec5-407f-b0d9-f8774760a05c.png)

To manage the teams treasury team task payments for March/April are being paid into two installaments. This issue documents the first installment of $5K USD. An additional $5K USD is due for March/April tasks and will be paid in June at the ADA-7DA rate at the time of payment. 

For installment 1, to help managing the available funds, some team members (@wolstaeb and @PardusEidolon) have elected to have payments deferred until June. The defered payments will be made at the ADA rate at the time of installment 1 payment as documented in this issue, rather than some later ADA-7DA conversion rate.

**Comment by wolstaeb** [comment]
First installment done, another $5KUSD is due and will be paid in June, in addition to the May payment.

**Comment by andrecahyadi** [comment]
@wolstaeb Hi Robert, how are you? :) 
I'd like to ask some questions regarding the task compensation sheet, https://docs.google.com/spreadsheets/d/1SHc84F-JPSyUSeylJ-JcfnxtqNUbYUZ0vNE97JF4g0c/edit#gid=0 :
* What does DRI stand for?
* How do you decide the apportion percentage? (cause it affects assignees' points) Why don't we just calculate based on assumptions only similar with DRI points?
* If every task is already decided by the "size", then it will be counted only after it has been 'Closed'? I see some tasks are not yet 'Closed', are those counted as well in the compensation? 
* About the exchange rate itself, any specific information when did you obtain these information? 
<img width="127" alt="image" src="https://user-images.githubusercontent.com/95466316/169699102-0b1702d4-5233-4d44-aa57-548056ac7a41.png">

**Comment by wolstaeb** [comment]
Hi @andrecahyadi thanks for these questions. 

Just a little context. This is the first time we have done task-based payments. People in the team need to get comfortable with writing up tasks as GH issues. Ideally they would do it before working on something but given the early stage of geting the compensation process underway I am not being strict. As we progress through each month we will get better at this.

The aim is to automate a big part of the task accounting via Github issues. But at this state we have to take baby steps. The [task compensation spreadsheet] was initially generated from the tasks listed in GH. And we will add more templates for common tasks to make the process easier.

> What does DRI stand for?

**DRI = Directly Responsible Individual.** This is the person taking the lead on making sure a task is done. Think of it like the person doing the project management. 

In many cases the DRI will be the same person doing the task but not always, and in the case of bigger efforts definitely not. For the March/April period the _DRI_ got %5 of the task/issue points attributed to all _Assignees_ (Individuals accountable for doing the work).

> How do you decide the apportion percentage?

By asking the inidivduals that worked on the task or the DRI since either had a better understanding of who did what. I started with everything set to 100% and then ask the team to adjust, then I went through and review it with the individuals concerned. 

In some cases, esp. meetings/workshops, people have assigned attendence as 100% for each individual, and the host is the DRI. The ID column indicates the tasks/issues that are the same.

> Why don't we just calculate based on assumptions only similar with DRI points?

Not sure what you mean here. Each task/issue is assigned a size. For the operations side of things we are using T-Shirt sizes and I have outlined some [rules-of-thumb for doing the sizings](https://github.com/C3ETH/c3eth-operations/wiki/Describing-Issue-Tasks). As we get better doing the task planning as a team our ability to estimate and sizes things will improve.

The task Tee-Shirt size then just maps that to points based on the fibonacci sequence (1,2,3,5,8) which aims to account for extra effort needed to do larger projects does not typically scale linearly.
 
> If every task is already decided by the "size", then it will be counted only after it has been 'Closed'?

Ideally yes. We will get there :)  However for the first iteration I just had to get everyone recording their tasks in some way. Ideally via individual GH issues. Most of the task recording was retrospective, not prospective. So I labelled all the tasks that where being considered in this round and closed them.

> I see some tasks are not yet 'Closed', are those counted as well in the compensation?

Not yet. They will go into the May task compensation.

> About the exchange rate itself, any specific information when did you obtain these information?

See above. It is documented in this issue above: "For Payment we use a simple seven day average to devise the payment conversion rates at the approximate time (within 30mins of) the payments being made...."

**Comment by andrecahyadi** [comment]
@wolstaeb Thank you for the explanation, Robert. Now I understand how to read the sheet. 
So, only DRI calculation that will be changing every 3 months?

**Comment by wolstaeb** [comment]
> Only DRI calculation that will be changing every 3 months?

We can change it for every period if we want. Once the team gets comfortable with the idea then we'll get better at setting the appropriate level. However before that happens people need to get more comfortable using GH Issues and Project Boards as a project planning tool. Then setting the appropriate level for the DRI will have more weight.

**Comment by wolstaeb** [comment]
@andrecahyadi I'm moving the [discussion to this thread](https://github.com/C3ETH/c3eth-operations/discussions/79)

**Comment by andrecahyadi** [comment]
> > Only DRI calculation that will be changing every 3 months?
> 
> We can change it for every period if we want. Once the team gets comfortable with the idea then we'll get better at setting the appropriate level. However before that happens people need to get more comfortable using GH Issues and Project Boards as a project planning tool. Then setting the appropriate level for the DRI will have more weight.

I see, noted.

**Comment by wolstaeb** [comment]
See issue #229 for the second installment.
